### PR TITLE
refactor: Switch to using bluebuild bin name

### PR DESCRIPTION
**Pull request by gmpinder**
This will need to be merged in at the same time as the next BlueBuild release

### CODE REVIEW

**Comment by gmpinder** [comment]
Will just need an approval from @xynydev and then after I'll coordinate merging this and releasing bluebuild v0.7.0

**Review by bayou-brogrammer — APPROVED** [review]

**Review by xynydev — APPROVED** [review]

**Comment by gmpinder** [comment]
Cool everything is approved. I'll work on getting the bin rename released today after work.

**Pull request merged**
